packages/tentales-log/src/index.ts

```diff
@@ -0,0 +1,25 @@
+// tslint:disable-next-line
+/// <reference path="./types/npmlog.d.ts" />
```

Comment:
Anyone knows of a better way to do this? 

There's no definition file available for npmlog, so this is my hacky way of creating a (non-complete) definition for TypeScript to be happy.

It would be nice if we could either

1. Avoid that `/// <reference`, or
2. Write the definition _inside_ this very file

Reply:
I thought `reference` wouldn't be needed there. Since you've declared the `npmlog` module in the `.d.ts` file 🤔

Might be that the ts-compiler needs to be told where to look for these typings?

Reply:
Yeah, I've been trying to get the TS compiler to pick up that file, but without success :/

To make things a bit more complicated, this is a mono-repo with a "master" tsconfig in its root and ~10 projects with their own tsconfigs (which inherits from the master one). And I want to be able to run it in both dev environment (where the project is run as-is, in typescript and not not compiled) and in production mode, where all these projects are their own npm modules, compiled to js.